framework/src/org/checkerframework/common/reflection/qual/ForName.java

```diff
@@ -0,0 +1,17 @@
+package org.checkerframework.common.reflection.qual;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * A @ForName annotation identifies the methods Class.forName and 
+ * ClassLoader.loadClass.
```

Comment:
Please copy comments from the arrays of ExecutableElement to the corresponding annotations.  
For example this annotation should have the comment: 
Annotations for methods of the form: @ClassVal("name") Class method(String name) {...}